### PR TITLE
net-proxy/mitmproxy: add 0.17.1 ebuild

### PR DESCRIPTION
**Pull request by andrey-utkin**

### CODE REVIEW

**Comment by andrey-utkin** [comment]
Will add -9999 build when I have time later.

File: net-proxy/mitmproxy/mitmproxy-0.17.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 versionator
+
+DESCRIPTION="An interactive, SSL-capable, man-in-the-middle HTTP proxy"
+HOMEPAGE="http://mitmproxy.org/"
+#SRC_URI="http://mitmproxy.org/download/${P}.tar.gz"
+SRC_URI="https://github.com/mitmproxy/mitmproxy/archive/v${PV}.tar.gz"
```
Comment (monsieurp) [inline net-proxy/mitmproxy/mitmproxy-0.17.1.ebuild]: v${PV}.tar.gz -> ${P}.tar.gz

**Comment by monsieurp** [comment]
There's a pending issue still. Please fix it so that we can move this forward. Thanks!

**Comment by monsieurp** [comment]
ping?

**Comment by radhermit** [comment]
If you happen to be partially pinging me, this still requires bumping all the rest of the mitmproxy suite deps as well as adding new rdep packages to the tree.

**Comment by andrey-utkin** [comment]
Sorry, got no time for this lately. This task is buried somewhere deeply in my TODO list.

**Comment by monsieurp** [comment]
Thanks @radhermit for the heads up. @andrey-utkin I'm going to close this PR since it appears to be incomplete. Feel free to file another one when you get round to it. Thank you for your input!

**Pull request closed without merging**